docs/vector_store.ipynb

```diff
@@ -1,79 +1,331 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @markdown Please fill in the value below with your Google Cloud project ID and then run the cell.\n",
+    "\n",
+    "PROJECT_ID = \"my-project-id\"  # @param {type:\"string\"}\n",
+    "\n",
+    "# Set the project id\n",
+    "!gcloud config set project {PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 💡 API Enablement\n",
+    "The `langchain-google-spanner` package requires that you [enable the Spanner API](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com) in your Google Cloud Project."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# enable Cloud SQL Admin API\n",
+    "!gcloud services enable spanner.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Cloud Spanner database values\n",
+    "Find your database values, in the [Cloud Spanner Instances page](https://console.cloud.google.com/spanner?_ga=2.223735448.2062268965.1707700487-2088871159.1707257687)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @title Set Your Values Here { display-mode: \"form\" }\n",
+    "INSTANCE = \"my-instance\"  # @param {type: \"string\"}\n",
+    "DATABASE = \"my-database\"  # @param {type: \"string\"}\n",
+    "TABLE_NAME = \"vectors_search_data\"  # @param {type: \"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialize a table\n",
+    "The `SpannerVectorStore` class instance requires a database table with id, content and embeddings columns. \n",
+    "\n",
+    "The helper method `init_vector_store_table()` that can be used to create a table with the proper schema for you."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_spanner import SpannerVectorStore, TableColumn, SecondaryIndex\n",
+    "\n",
+    "SpannerVectorStore.init_vector_store_table(\n",
+    "    instance_id=INSTANCE,\n",
+    "    database_id=DATABASE,\n",
+    "    table_name=TABLE_NAME,\n",
+    "    id_column=\"row_id\",\n",
+    "    metadata_columns=[\n",
+    "        TableColumn(name=\"metadata\", type=\"JSON\", is_null=True),\n",
+    "        TableColumn(name=\"title\", type=\"STRING(MAX)\", is_null=False),\n",
+    "    ],\n",
+    "    secondary_indexes=[\n",
+    "        SecondaryIndex(index_name=\"row_id_and_title\", columns=[\"row_id\", \"title\"])\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Embedding Models\n",
```

Comment:
nit: point to VertexAIEmbeddings like https://github.com/googleapis/langchain-google-cloud-sql-pg-python/blob/main/docs/vector_store.ipynb?short_path=5c80901#L281